proposals/dark_mode_extension_icons.md

```diff
@@ -0,0 +1,158 @@
+# Dark Mode Extension Icons
+
+**Summary**
+
+Feature to enable developers to enhance extension icon visibility in dark mode.
+
+**Document Metadata**
+
+**Author:** &lt;solomonkinard>
+
+**Sponsoring Browser:** Chromium.
+
+**Contributors:** &lt;oliverdunk>
+
+**Created:** 2024-04-05
+
+**Related Issues:** &lt;https://crbug.com/893175>
```

Comment:
The link is broken because of `>` (you should use the markdown link format). Also, I suggest adding the WECG link #229 here.

Reply:
Addressed in the patch response.